.jenkins/pipelines/p1-tests.Jenkinsfile

```diff
@@ -16,7 +16,6 @@ pipeline {
         booleanParam(name: "RUN_DOTNETP1", defaultValue: true, description: "Run .NET P1 tests?")
         booleanParam(name: "RUN_OPENMP_TESTSUITE", defaultValue: true, description: "Run OpenMP Test Suite?")
         booleanParam(name: "ICELAKE_VM", defaultValue: true, description: "Run tests on Ice Lake VMs?")
```

Comment:
You can remove this parameter too since this must be true (if the run were to do anything).